src/Indicator.vala

```diff
@@ -126,6 +134,25 @@ public class Session.Indicator : Wingpanel.Indicator {
             }
 
             main_grid.add (suspend);
+
+            try {
+                if (suspend_interface.can_hibernate () == "yes") {
+                    main_grid.add (hibernate);
+                }
+            } catch (GLib.Error e) {
+                warning ("CanHibernate Error: %s", e.message);
+                hibernate.set_sensitive (false);
```

Comment:
Not sure it makes sense to set this widget as insensitive if it's never packed in the first place